Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert_v2.yml

```diff
@@ -0,0 +1,1104 @@
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Convert an array to a nice table display. Usually, from the context.
+      id: de3784fe-7b21-4ebc-87fa-76c3183fe4f1
+      iscommand: false
+      name: Set Audits to Table
+      script: ToTable
+      type: regular
+      version: -1
+    taskid: de3784fe-7b21-4ebc-87fa-76c3183fe4f1
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 865
+        }
+      }
+  "42":
+    continueonerrortype: ""
+    id: "42"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "26"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 799d7dec-7bfd-42ad-8cf6-9c8f85049299
+      iscommand: false
+      name: Manually Remediate
+      description: Perform the manual remediation
+      type: regular
+      version: -1
+    taskid: 799d7dec-7bfd-42ad-8cf6-9c8f85049299
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 4420
+        }
+      }
+  "43":
+    continueonerrortype: ""
+    id: "43"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "11"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      extend-context:
+        simple: 'Indicators=.={"Value": val.value, "Description": val.CustomFields.description,
+          "CVSS": val.CustomFields.cvss}'
+      ignore-outputs:
+        simple: "true"
+      query:
+        simple: incident.id:${incident.id} and type:CVE
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Builtin
+      description: commands.local.cmd.find.indicators
+      id: 87c58322-6643-47db-85e7-4976bdc771b5
+      iscommand: true
+      name: Create Indicator Table
+      script: Builtin|||findIndicators
+      type: regular
+      version: -1
+    taskid: 87c58322-6643-47db-85e7-4976bdc771b5
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 4100
+        }
+      }
+  "44":
+    continueonerrortype: ""
+    id: "44"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "45"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      imageName:
+        simple: '${PrismaCloudCompute.AuditAlert.image} '
+      type:
+        simple: '${PrismaCloudCompute.AuditAlert.category} '
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 69a6d29a-a700-4b45-8336-2bbfb6bab7a0
+      iscommand: false
+      name: Create Link Back
+      description: Create a link back to the Prisma Cloud Compute instance.
+      script: 1c969de3-c684-4977-872b-295ad7be3f0b
```

Comment:
in this task you use the `CreatePrismaCloudComputeLink` script
please use the HttpV2 script instead and try to use in without hardcoded values

Reply:
whoops! yes will fix the hardcoded values in `CreatePrismaCloudComputeLink`. This doesn't actually make the HTTP request to the URL it just creates the format of the link back to alerts for the image in Prisma Cloud Compute. Will fix this one.

Reply:
in that case please create a task in the playbook to create the url using **concat** transformer